src/components/widgets/Mo2PercentWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {IPoint} from 'types/IPoint';
+import {BarChart} from "components/widgets/shared/BasicBarGraph";
```

Comment:
Should call the file “BasicBarChart”, and BarChart -> BasicBarChart to be consistent

Reply:
Changed all the names to be BasicBarChart